TLM/TLM/Manager/Impl/SpeedLimitManager.cs

```diff
@@ -301,10 +301,22 @@ public class SpeedLimitManager
                 defaultValue: null);
         }
 
-        public float GetLockFreeGameSpeedLimit(ushort segmentId,
-                                               byte laneIndex,
-                                               uint laneId,
-                                               NetInfo.Lane laneInfo) {
+        float ISpeedLimitManager.GetGameSpeedLimit(uint laneId) {
```

Comment:
you have implemented the interface explicitly which is useful to resolve ambiguity for overloaded methods. 
this code would work. I am not an expert but I think `ISpeedLimitManager.`  part is redundant. 
the rest of the TMPE does not implement interface explicitly so I suggest to follow the general style.

Reply:
No need to use explicit interface implementation here.
The only reason for explicit interface implementation is when class implements multiple interfaces and more than one interface has methods with the same signature.